docs/source/verticapylab_gs.rst

```diff
@@ -0,0 +1,141 @@
+.. _verticapylab_gs:
+
+=================
+VerticaPyLab
+=================
+
+The easiest way to try out VerticaPy is with `VerticaPyLab <https://github.com/vertica/VerticaPyLab>`_, a docker-based 
+JupyterLab environement designed to provide users with a powerful, easy-to-use development platform for VerticaPy. Capitalizing
+on the convenicnece of docker, VerticaPyLab eliminates the need for any intricate installations, configurations, and dependency 
+management. VerticaPyLab extends the capabilities of JupyterLab by integrating essential tools, extensions, and utilities that 
+allow you to unlock the full potential of VerticaPy's machine learning and analytic funtionalities, all while utilizing the 
+performance and scalability of the Vertica columnar database. 
+
+VerticaPyLab utilizes two containers, one housing a Vertica database and the other a JupyterLab container with VerticaPy and 
+necessary dependecies installed. Using these two components, VerticaPyLab offers a cohesive environment that bridges the power 
+of Vertica with the capabilities of Python's ML toolkit.
+
+Some key features of VerticaPyLab include the following:
+- Autocompletion and syntax highlighting: enhance your coding efficiency and write more accurate, error free code with the 
```

Comment:
I am not sure what you mean here. VerticaPyLab does not include either syntax highlighting or autocompletion right now. Or maybe I am not understanding this point well.

Reply:
I borrowed this idea from an existing confluence page, but maybe that was a page that included future functionality. Will remove.